source/getting-started.txt

```diff
@@ -0,0 +1,135 @@
+.. _app-services-get-started:
+
+================
+Getting Started
+================
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   
+   Tutorials </tutorials>
+   Template Apps </template-apps>
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _app-services-get-started-overview:
+
+The resources on this page are designed to help you explore Atlas App 
+Services. To explore a working codebase that showcases App Services' 
+features, check out a Template App. For a more guided experience in
+learning how to develop with App Services and Realm, start with a 
+Tutorial.
+
+.. _app-services-get-started-template-apps:
+
+Template Apps
+-------------
+
+Bootstrap App Services with a :doc:`Template App </template-apps>`. Template apps bring together
```

Comment:
please replace the `:doc:` directive with a `:ref:`

Reply:
Getting a snooty error when I try to change to a :ref:, would.a :doc: be fine as is?